src/components/Books/AddBook.js

```diff
@@ -3,21 +3,21 @@ import React from 'react';
 import { useDispatch } from 'react-redux';
 import { v4 as uuidv4 } from 'uuid';
 
-import { addBook } from '../../redux/books/books';
+import { addBookApi } from '../../redux/books/books';
 
 const AddBook = () => {
   const dispatch = useDispatch();
   const submitBookToStore = (e) => {
     e.preventDefault();
     const newBook = {
-      id: uuidv4(),
+      item_id: uuidv4(),
       title: e.target.title.value,
-      author: e.target.author.value,
+      category: e.target.category.value,
     };
 
-    dispatch(addBook(newBook));
+    dispatch(addBookApi(newBook));
     e.target.title.value = '';
-    e.target.author.value = '';
+    e.target.category.value = '';
```

Comment:
- [ ] I can add empty book without title or category. It would be nice to validate book data before creating a new book.